### PR TITLE
Dynamic mode turndown

### PR DESCRIPTION
**Pull request by ShiftyRail**
Dynamic is a wonderful piece of code, don't get me wrong, but I think it currently puts too much antags in the round too early.
This PR ups the threat cost of changelings, midround wiz, and make it slightly less likely to happen.

This also adds an "extended" roundstart ruleset, with a slightly low chance of getting picked.
The threat requirement and cost is null, so roundstart extended is more likely to turn into late nukies, for instance.

Tested

:cl:
- rscadd: Re-adds extended as a possible roundstart ruleset.
- rscadd: Ups the threat level of changelings & wizards. Effectively, you should see them slightly less often.

### CODE REVIEW

**Comment by Accipitermaris** [comment]
I could have sworn the original design doc listed "Peaceful" at the bottom of threat rolls but it had like a 1% chance to roll
Anyway, good change

**Comment by DeityLink** [comment]
>Extended ruleset

like, the requirements var exists partly so we wouldn't need that

Other than that I'm ok with the changes.

@Accipitermaris It's basically implemented already, if the server rolls under 10% threat level there won't be any antags. But even Traitor which requires the less threat will require more than 10% if there are little players on the server. Adding extended as a ruleset goes against everything dynamic mode stood for.

**Comment by DeityLink** [comment]
Also another sensible reason to let threat level and requirements do their job instead of adding an extended ruleset:

>there's enough threat and pop for 2 roundstart rulesets
>rolling extended and wizard
>"starting a round of extended....and also a wizard"

TL;DR, if you want less antags in the round early, up the costs (which you already did), and increase the required threat level per pop range.
For instance, with changeling, replace requirements = list(80,60,40,20,20,10,10,10,10,10) with requirements = list(90,70,50,30,20,10,10,10,10,10)

**Comment by ShiftyRail** [comment]
Oh you're right

**Comment by ShiftyRail** [comment]
I'll explain in more details why I think something like an extended rule to pick is important.
>there's enough threat and pop for 2 roundstart rulesets
rolling extended and wizard
"starting a round of extended....and also a wizard"

I agree that this makes the message non-descriptive, but this replace, says, wiz + another antag at roundstart.
It also allows for extended to proc naturally in lowpop (instead of, say, lowpop vampire), and to be used as a forced roundstart ruleset.

I see an extended ruleset as 'blank' rule that doesn't costs any threat so you can ideally keep threat for later antag injections (which are, alas, currently broken).

**Comment by DeityLink** [comment]
>to be used as a forced roundstart ruleset.

That's the only valid reason for it to exist to me.

>instead of, say, lowpop vampire

Just up the requirements for lowpop vampire, duh

>I see an extended ruleset as 'blank' rule that doesn't costs any threat

Well except even with midround antags fixed you're denying antags that can only spawn at roundstart.

You're bringing a solutions to problems that don't exist, and those who do exist have solutions already built in for them.

**Comment by iFlashYou** [comment]
merge when?
